### PR TITLE
added my name

### PR DESCRIPTION
**Pull request by ayuushhh**
my first pr

### CODE REVIEW

**Comment by AarKiMos** [comment]
you missed a comma at line 22, also you need to add all the lines as required,if you do not want to add any detail,leave the quotes empty. If you wish to get this PR merged, make corrections and commit the changes to your branch. New commits will be added to this PR automatically.

**Comment by ayuushhh** [comment]
i have made all the necessary corrections.please check it

File: src/Contributors.js [hunk 1]
```diff
@@ -20,4 +20,10 @@ export default [
   	facebook: "",
   	twitter: "aaryan7476",
   }
```
Comment (AarKiMos) [inline src/Contributors.js]: @ayuushhh  As you can see all the checks have failed. As a hint, there is a small error at line 22. Try to find the error yourself this time.

**Review by AarKiMos — CHANGES_REQUESTED** [review]

File: src/Contributors.js (hunk 1 above)
Comment (ayuushhh) [reply src/Contributors.js]: Sir i corrected the mistakes .Please check it.

**Pull request merged**